### DISCUSSION TITLE
Is there a plan to support cross-thread trace for ForkJoinPool?

### DISCUSSION THREAD

**Discussion by darknesstm** [apache/skywalking#10673] [General]
`apm-jdk-threadpool-plugin` is not supported.

**Comment by wu-sheng** [comment]
Usually, we only accept plugins if someone wants to contribute. The maintainers are not adding new plugins unless they need by themselves.

**Comment by wu-sheng** [comment]
Added through https://github.com/apache/skywalking-java/pull/505, planned in next release(July routine agent release).
